examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>I think "Applying to your own dataset" has a higher motivation factor for users to read the section than "integration example"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I agree "Applying to your own dataset" caches more the attention of the user. I just suggested the change bcs as I user, from that title I would hope a notebook with placeholders to provide my data path and some config option to run easily ETL on my own dataset, like in this [example notebook (section Config)](https://github.com/rapidsai/recsys/blob/main/recsys_dataset_profiles/template_recsys_dataset_profiler.ipynb).&nbsp;</p><p> </p><p>I think in this notebook we use movielens to demo the integration between NVTabular and Models using schema which is great, but the notebook does make it straightforward applying it to your own dataset, IMHO.</p><p>It is up to you if my point makes sense (and updating it to Integration example). If not I can return the previous title, just let me know your thoughts on that.</p>

Reply:
<p>I understand the point - but the title does not communicate why this example is important for a user. A user do not know what Merlin / NVTabular really is and why the integration is important?</p>

Reply:
I have changed the example title to "From ETL to Training RecSys models - NVTabular and Merlin Models integrated example" to highlight this example is about understanding the full preprocessing and training pipeline